js/text-edit.picker.js

```diff
@@ -97,7 +129,7 @@ var markdownEditor = {
 
         xhr.open("PUT", url, true);
         xhr.setRequestHeader("Content-type", "application/octet-stream");
-        xhr.setRequestHeader("Authorization", "Bearer " + this.accessToken)
+        xhr.setRequestHeader("Authorization", "Bearer " + parent.accessToken.accessToken)
```

Comment:
I used to just store the string, but this update changes it to store more properties about the token, since we need them.